src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -6,6 +6,7 @@ IntakeSubsystem::IntakeSubsystem() {
 
 void IntakeSubsystem::Periodic() {
   // intakeTuner.UpdateFromShuffleboard();
+  ShuffleboardLogger::getInstance().logVerbose("Has Note", NotePresent());
```

Comment:
Don't use the logger but rather write directly using `SmartDashboard::PutBoolean`